server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala

```diff
@@ -25,11 +25,11 @@ import sttp.tapir.server.interceptor.reject.RejectInterceptor
 import sttp.tapir.server.interpreter.{BodyListener, FilterServerEndpoints, ServerInterpreter}
 import sttp.tapir.server.model.ServerResponse
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 trait AkkaHttpServerInterpreter {
 
-  def akkaHttpServerOptions: AkkaHttpServerOptions = AkkaHttpServerOptions.default
+  def akkaHttpServerOptions: AkkaHttpServerOptions
```

Comment:
maybe this should be done as in `PlayServerInterpreter`, with an EC as an implicit parameter in the trait, to maintain consistency across server interpreters

Reply:
Done